tools/internal_ci/linux/grpc_aws_experiment.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script is responsible for remotely running tests on an ARM instance.
+# It should return a status code useful to the kokoro infrastructure.
+# It currently assumes an instance will be selected by the time this script begins running.
+
+if [ -z "$KOKORO_KEYSTORE_DIR" ]; then
+    echo "KOKORO_KEYSTORE_DIR is unset. This must be run from kokoro"
+    exit 1
+fi
+
+IDENTITY=${KOKORO_KEYSTORE_DIR}/73836_grpc_arm_instance_ssh_private_test_key1
+
+if [ -z "$INSTANCE" ]; then
+    echo "no INSTANCE set, aborting"
+    exit 1
+fi
+
+FILE=grpc_aws_experiment_remote.sh
+chmod 700 $IDENTITY
+ssh -i $IDENTITY -o StrictHostKeyChecking=no ubuntu@$INSTANCE "rm -rf grpc"  # TODO: remove when using fresh instances
```

Comment:
nit: I think it would be handy to put all the files in a "workspace" subdirectory, just for separation.
e.g the script could create "~/remote_executor_workspace" directory, which would make it easier to keep all the files contained and also to support other codebases than just grpc.

Reply:
I can create another directory if you'd like. Ultimately, however, each vm will be used once. In this case, I'm not sure what the difference is. Would we ever check out multiple codebases in a single test run?

Reply:
I'd prefer to have that directory. I agree that currently it's not strictly required (since we are going to be throwing away the VMs), but in the future it might come handy. It might also make logs more digestible since the purpose of that directory is obvious just from its name.